File: src/Java/Store/5. Final Hole/src/main/refactoringgolf/store/AccessoriesDiscount.java
package refactoringgolf.store;

public class AccessoriesDiscount implements CategoryDiscount{

	public float calculateDiscount(OrderItem orderItem) {
		float discount = 0;
		if (orderItem.getUnitPricePerQuantity() >= 100) {
			discount = orderItem.getUnitPricePerQuantity() * 10 / 100;
		}
		return discount;
	}

}

File: src/Net/1. First Course/3. Third Hole/RefactoringGolf.Store/AccessoriesDiscount.cs
namespace RefactoringGolf.Stack
{
    public class AccessoriesDiscount : ICategoryDiscount
    {
        public decimal CalculateDiscount(OrderItem orderItem)
        {
            decimal discount = 0;
            if (orderItem.ItemAmount() >= 100)
            {
                discount = orderItem.ItemAmount() * 10 / 100;
            }
            return discount;
        }
    }
}